beacon_node/network/src/service.rs

```diff
@@ -789,32 +785,29 @@ impl<T: BeaconChainTypes> NetworkService<T> {
         }
     }
 
+    /// Keeping these separate from core topics because it has custom logic:
+    /// 1. Data column subscription logic depends on subscription configuration.
+    /// 2. Data column topic subscriptions will be dynamic based on validator balances due to
+    ///    validator custody.
```

Comment:
The downside with not including it in core fork topic is - we subscribe to PeerDAS topics as soon as Fulu is scheduled, rather than waiting until the fork.

We could potentially just duplicate the logic here:
https://github.com/sigp/lighthouse/blob/770d677a4e25df54f87789b0f6269c4be1149f96/beacon_node/lighthouse_network/src/service/mod.rs#L708

Reply:
You should add a TODO for this

Reply:
Existing issue for this #5894